Source/MediaInfo/Multiple/File_P2_Clip.h

```diff
@@ -47,7 +46,7 @@ private :
 
     //Buffer - File header
     bool FileHeader_Begin();
-

```

Comment:
just update the PR with old commit removed and new commit (but if you don't know how to do it, I may accept a new PR)